README.md

```diff
@@ -72,6 +72,30 @@ Log out from postgres and run:
 
 Your server is ready to run. You can do this by executing `rails server` and going to [http://localhost:3000](http://localhost:3000). Happy coding!
 
+## AngularJS Guide
+
+AngularJS is added inside `vendor/assets/javascripts` folder. The current version is `v1.6.3  `.
```

Comment:
Ok. I read the documentation better and found this:
```
Every attempt is made to mirror the currently shipping Angular.js version number wherever possible.
The major, minor, and patch version numbers will always represent the Angular.js version.
```
So we should fix Gems version to the desired Angular version.